icons/gitlfs.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Git LFS icon</title><path d="M0 16.82l12.04 6.948v-9.675L24 7.184l-4.632-2.676-11.96 6.907V15.3l-2.776-1.601V9.813l11.965-6.908L11.966.233 0 7.14zm12-2.79l.04 9.738L24 16.858V7.185z"/></svg>
```

Comment:
This matches perfectly! 💯
One thing I'd suggest is if possible, try and remove any internal points that don't add anything to the icon (see below). I'm also going to ping @simple-icons/maintainers, as while the monochrome treatment here is great, I would like another opinion as to whether the colour change should be outlined or not.
![image](https://user-images.githubusercontent.com/1830867/104453440-525cba80-559c-11eb-8bb2-10806e8a9e49.png)

Reply:
I would prefer a gap (0.5px) where the color changes, conform to our [contribution guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md).

> Some icons can't be easily converted to a monochromatic version due to colour changes, shadows, or other effects. For such cases the addition of gaps is the recommended approach, with a preferred width of 0.5px. In some situations a different gap may be required, but that will be determined on a per-case basis.

@loganmarchione Just to give you an idea, it should look like this in my opinion:

![image](https://user-images.githubusercontent.com/70264417/104455719-fc8e0f80-55a7-11eb-8bf3-69abced4c7b5.png)


### How to achieve it in Inkscape (if you use that?)
- Draw a Bezier curve (shortcut: B) along the edges of the box.
- Set the fill to none, the stroke to a width of 0.5px. (while the icon itself has a width/heigth of 24px)
- Convert the stroke to a path.
- Create the difference between the gap and the box (Ctrl + -).

Reply:
@adamrusted 
Ah ok, I didn't think about that. First, here is the shape with that extra corner removed.
`<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Git LFS icon</title><path d="M0 16.82l12.04 6.948L24 16.858V7.184l-4.632-2.676-11.96 6.907V15.3l-2.776-1.601V9.813l11.965-6.908L11.966.233 0 7.14z"/></svg>`

Here is the 2nd one (with the gap). Let me know if it looks ok and I can add a new commit.
`<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Git LFS icon</title><path d="M11.967.232L0 7.141v9.68l11.79 6.802v-9.674l11.962-6.91-4.385-2.531-11.959 6.906v3.887l-2.775-1.602V9.813l11.965-6.909L11.967.232zM23.75 7.62l-11.46 6.62v9.095l11.46-6.621V7.619z"/></svg>`

Reply:
Ah ok sorry I drew the box around all 4 edges. Try this.
`<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Git LFS icon</title><path d="M11.967.232L0 7.141v9.68l11.79 6.802v-9.674l11.96-6.91-4.383-2.531-11.959 6.906v3.887l-2.775-1.602V9.813l11.965-6.909L11.967.232zM24 7.472l-11.71 6.764v9.387L24 16.857V7.473z"/></svg>`

Reply:
Perfect @loganmarchione! Would you mind pushing that version to this PR and we can merge it!